src/skills-builder/skills-builder-steps/select-preferences/CareerInterestCategorizinator.jsx

```diff
@@ -9,7 +9,7 @@ import CareerInterestCard from './CareerInterestCard';
 import { addCareerInterest, clearAllCareerInterests, setExpandedList } from '../../skills-builder-context/data/actions';
 import { SkillsBuilderContext } from '../../skills-builder-context';
 import messages from './messages';
-import { careerList } from '../../utils/jobsByCategory';
+import careerList from '../data/jobsByCategory.json';
```

Comment:
Nice. I was getting all sorts of warnings about having to assert it as JSON when I tried to do that.